opencti-platform/opencti-front/src/private/components/data/Feed.tsx

```diff
@@ -118,7 +120,9 @@ const Feed = () => {
       <Breadcrumbs variant="list" elements={[{ label: t_i18n('Data') }, { label: t_i18n('Data sharing') }, { label: t_i18n('CSV feeds'), current: true }]} />
       <SharingMenu/>
       {feedState.view === 'lines' ? renderLines(paginationOptions) : ''}
-      <FeedCreation paginationOptions={paginationOptions}/>
+      <Security needs={[TAXIIAPI_SETCOLLECTIONS]}>
```

Comment:
this doesn't seem right, because Feed is a generic component used for all feeds (CSV feeds &Taxii collections)